pkg/parameters/parameterstore.go

```diff
@@ -0,0 +1,86 @@
+package parameters
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/cnabio/cnab-go/utils/crud"
+)
+
+// TODO: very much a clone of cnab-go/credentials/credstore.go
+// Ideas on how to generalize for re-use?
+// Perhaps update methods to all take an additional ItemType param?
+
+// ItemType is the location in the backing store where parameters are persisted.
+const ItemType = "parameters"
+
+// ErrNotFound represents a parameter set not found in storage
+var ErrNotFound = errors.New("Parameter set does not exist")
+
+// Store is a persistent store for parameter sets.
+type Store struct {
+	backingStore *crud.BackingStore
+}
+
+// NewParameterStore creates a persistent store for parameter sets using the specified
+// backing key-blob store.
+func NewParameterStore(store crud.Store) Store {
```

Comment:
Are there unit tests for this struct?

Reply:
😅  Not yet.  Workin' on some for this as well as `parameterStorage.go`  Thanks!

Reply:
Unit tests added!